### ISSUE TITLE
Master build is failed

### ISSUE THREAD

**Issue by rdiachenko** [checkstyle/patch-filters#42]
Steps to reproduce:
```
$ git clone git@github.com:checkstyle/patch-filters.git
$ cd patch-filters
$ mvn clean install
```
Detected:
```
Results :

Failed tests:   testBoundaryOne(com.github.checkstyle.patchfilter.SuppressionPatchFilterTest): Audit event should be rejected when there are no matching patch filters ==> expected: <true> but was: <false>
  testMultiChangedFilesOnOnePatch(com.github.checkstyle.patchfilter.SuppressionPatchFilterTest): Audit event should be rejected when there are no matching patch filters ==> expected: <true> but was: <false>

Tests run: 14, Failures: 2, Errors: 0, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.495 s
[INFO] Finished at: 2020-06-25T20:16:36+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.12.4:test (default-test) on project patch-filter: There are test failures.
```
Expected:
```
Build success
```

Notes:
```
$ java -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
```

**Comment by HuGanghui** [comment]
@rdiachenko this problem will be fixed when https://github.com/checkstyle/patch-filters/pull/41 is merged

**Comment by rdiachenko** [comment]
Fix was merged in #41
